src/etl/LoadBalancer.cpp

```diff
@@ -84,6 +91,27 @@ LoadBalancer::LoadBalancer(
     std::shared_ptr<NetworkValidatedLedgersInterface> validatedLedgers,
     SourceFactory sourceFactory
 )
+    : forwardedDurationHistogram_(PrometheusService::histogramInt(
+          "lb_forwarded_duration_milliseconds_histogram",
+          Labels(),
+          kHISTOGRAM_BUCKETS,
+          "The duration of processing forwarded requests"
+      ))
+    , forwardedRetryCounter_(PrometheusService::counterInt(
+          "lb_forwarded_retry_counter",
```

Comment:
I think `lb` prefix is not very discoverable, `load_balancer` or even `forwarding` prefix would be better I think.